src/main/java/com/mlaf/hu/decisionagent/behaviour/RegisterSensorAgentBehaviour.java

```diff
@@ -32,8 +32,12 @@ public void action() {
             String responseContent = is.checkComposition();
             ACLMessage response = new ACLMessage(ACLMessage.DISCONFIRM);
             response.setContent(responseContent);
+            if (DA.sensorAgentExists(message.getSender())) {
+                response.setContent("You are already registered.");
```

Comment:
Moet dit niet in XML format zijn? of als hij zijn informatie uit de performative haalt, heeft deze content uberhaubt zin?

Reply:
Ik heb het weggehaald. Ik besefte me dat het niet echt een GUI applicatie is voor normale gebruikers haha. Kortom geen beschrijving needed, maar alleen de performative CONFIRM. Thanks.